app/views/admin/creator_settings/new.html.erb

```diff
@@ -1,48 +1,47 @@
 <% if FeatureFlag.enabled?(:creator_onboarding) && User.with_role(:creator).any? %>
-<style>
-  <%= Rails.application.assets["setup-mode.css"].to_s.html_safe %>
-  #page-content {
-    display: flex;
-    flex-direction: column;
-  }
-</style>
-<main id="main-content" class="flex flex-1 justify-center flex-col crayons-layout crayons-layout--limited-xs">
-  <div aria-live="assertive">
-    <% if flash[:error] %>
-      <div class="crayons-notice crayons-notice--danger mb-6" role="alert">
-        <%= flash[:error] %>
-      </div>
-    <% end %>
-  </div>
+  <style>
+    <%= Rails.application.assets["setup-mode.css"].to_s.html_safe %>
```

Comment:
Refactor (potentially blocking): From what I read, `Rails.application.assets` is always `nil` in production because the assets are precompiled. It works in development because `config.assets.compile = false`. For production, i.e. precompiled assets, `Rails.application.assets_manifest.assets[path]` appears to be the way to go.

Reply:
I was mostly following the pattern that exists in our app, and since I was not able to find a reasonable answer, I asked for clarity [here](https://forem-team.slack.com/archives/C7GE84DEJ/p1636393617356200).

Reply:
I'm going to go ahead and approve and if @benhalpern suggests a better way to inline this, we'll do that, otherwise, I guess keep as is.